File: M3104/tomcat-etud/webapps/gpsapp/WEB-INF/src/db/Journey.java
package db;


import java.sql.Date;

public class Journey {
	private int id;
	private String description;
	private Date date_creation;
	private float distance;

    public Journey(String description) {
        this.id = id;
        this.description = description;
        this.date_creation = date_creation;
        this.distance = distance;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Date getDate_creation() {
        return date_creation;
    }

    public void setDate_creation(Date date_creation) {
        this.date_creation = date_creation;
    }

    public float getDistance() {
        return distance;
    }

    public void setDistance(float distance) {
        this.distance = distance;
    }

    @Override
    public String toString() {
        String ret = "--- Journey ---";
        ret += "\nID = " + id;
        ret += "\nDESCRIPTION = " + description;
        ret += "\ndate_creation = " + date_creation.toString();
        ret += "\nDISTANCE = " + distance;
        ret += "\n---------------\n";
        return ret;
    }
}

File: M3104/PHP/my_gps_web_app/model/Journey.php
<?php

class Journey {
    private $id;
    private $description;
    private $date_creation;
    private $distance;

    public function getId() {
        return $this->id;
    }

    public function setId($newId) {
        $this->id = $newId;
    }

    public function getDescription() {
        return $this->description;
    }

    public function setDescription($newDescription) {
        $this->description = $newDescription;
    }

    public function getDateCreation() {
        return $this->date_creation;
    }

    public function setDateCreation($newDate_creation) {
        $this->date_creation = $newDate_creation;
    }

    public function getDistance() {
        return $this->distance;
    }

    public function setDistance($newDistance) {
        $this->distance = $newDistance;
    }
}

?>
